### ISSUE TITLE
a11y: 'Value' textfield in authorizations modal is not associated with label

### ISSUE THREAD

**Issue by nwpappas** [swagger-api/swagger-ui#8879]
### Q&A (please complete the following information)
 - OS: all
 - Browser: all

### Describe the bug you're encountering
Label 'Value' is not associated with their respective form field. Screen reader only announces it 'Edit blank'.

### To reproduce...

1. Open URL: https://petstore.swagger.io/
1. Navigate on 'Authorize' button
1. 'Available authorizations' section appeared. Navigate on 'Value' edit field
1. Press F12 to open the developer tool
1. Verify that Label 'Value' should be associated with their respective form field or not.

### Expected behavior
Label 'Value' should be associated with their respective form field.

### Screenshots
![image](https://github.com/swagger-api/swagger-ui/assets/2529208/5043e688-a229-4713-b816-9750cb65038b)

### Additional context or thoughts
In the absence of label information for form elements, then users dependence with screen reader are not able to understand the purpose of UI component and could give-up to use the functionality.

**Comment by glowcloud** [comment]
Addressed in https://github.com/swagger-api/swagger-ui/pull/9599
